apps/admin_api/lib/admin_api/v1/controllers/api_key_controller.ex

```diff
@@ -54,12 +54,7 @@ defmodule AdminAPI.V1.APIKeyController do
   @spec create(Plug.Conn.t(), map()) :: Plug.Conn.t()
   def create(conn, attrs) do
     with {:ok, _} <- authorize(:create, conn.assigns, attrs),
-         # Admin API doesn't use API Keys anymore. Defaulting to "ewallet_api".
-         {:ok, api_key} <-
-           APIKey.insert(%{
-             owner_app: "ewallet_api",
-             originator: Originator.extract(conn.assigns)
-           }),
+         {:ok, api_key} <- APIKey.insert(%{originator: Originator.extract(conn.assigns)}),
```

Comment:
`APIKey` now has a `name` field that can be defined, so we need to get it from the `attrs` map.